libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -586,11 +636,37 @@ void AP_Baro::update(void)
         }
     }
 
+    // consider a sensor as healthy if it has had an update in the
+    // last 0.5 seconds
+    uint32_t now = AP_HAL::millis();
+    for (uint8_t i=0; i<_num_sensors; i++) {
+        sensors[i].healthy = (now - sensors[i].last_update_ms < 500) && !is_zero(sensors[i].pressure);
```

Comment:
The backends already do this directly, why is it needed here?

Reply:
See above.